eo-parser/src/test/java/org/eolang/parser/PacksTest.java

```diff
@@ -41,15 +36,9 @@
 final class PacksTest {
 
     @ParameterizedTest
-    @MethodSource("yamlPacks")
+    @ClasspathSource(value = "org/eolang/parser/packs/", glob = "**/*.yaml")
     void testPacks(final String pack) throws Exception {
```

Comment:
@yegor256  `testPacks` -> `parsesPacks`

Reply:
@volodya-lombrozo I didn't touch the names of methods, just modified their implementation, that's why... I will modify them now